### DISCUSSION TITLE
how similar is this to bullet?

### DISCUSSION THREAD

**Discussion by vhollander** [jrouwe/JoltPhysics#1234] [Q&A]
hi,

i've tried the engine out last year and it's impressive how it works efficiently. i've recently read PhysicsSystem.h and assumed its quite similar to bullet's btDiscreteDynamicsWorld.h at first glance. i'd like to make a bullet->jolt wrapper to benefit from its performance without major userside code refactoring, what should i do? 

thanks!

**Comment by DocAce** [comment]
I switched our game from Bullet to Jolt a while back and had to refactor a bunch of things in our wrapper, for example collision masks, filters and how bodies are created. While you can probably create a wrapper that can handle both under the hood, its interface is almost guaranteed to change in some key ways.
